Feat/agen mode

IgnatovFedor opened the pull request.

IgnatovFedor committed:
feat: added agent mode

Now we can send data to

IgnatovFedor committed:
docs: added correct CRBP launch instruction

litinsky reviewed (changes requested):
Add send json documentation to `--state`, `--agent`, `--convai` modes and their all possible combinations.

IgnatovFedor committed:
refactor: added named tuple for Message

Added description to documentation and added console argument check: blocked setting convai and state modes simultaneously with agent mode

IgnatovFedor committed:
docs: changed example states
